addon/-private/is-rfc268-test.js

```diff
@@ -0,0 +1,37 @@
+import { getContext, visit } from './rfc268-helpers';
+
+export default function isRfc268Test() {
+  // `getContext()` returns:
+  //  - falsey, if @ember/test-helpers is not available (stubbed in
+  //    rfc268-helpers.js)
+  //  - falsey, if @ember/test-helpers is available but none of the
+  //    `ember-qunit` setupTest() methods has been called (e.g.,
+  //    `setupRenderingTest()`)
+  //  - truthy, if @ember/test-helpers is available and one of the `ember-qunit`
+  //    setupTest() methods has been called.
+  //
+  // Note that if `page.setContext(this)` has been called, we'll never get here
+  // and will just be running with the integration context (even if the test is
+  // an RFC268 test).
+  let hasValidTestContext = Boolean(getContext());
+  if (!hasValidTestContext) {
+    return false;
+  }
+
+  // There are a few versions of `@ember/test-helpers` that have support for
+  // `ember-qunit`'s `setupRenderingTest()` method, but do not have the DOM
+  // helpers (`click`, `fillIn`, etc.) that the RFC268 execution context uses.
+  // `visit` was the last helper to be added to `@ember/test-helpers`, so we
+  // check for it, and if we can't find it, we can't use the RFC268 execution
+  // context, so we throw an exception.
```

Comment:
Again, super helpful in explaining hard-to-discover subtleties/background information.